config/samples/otel.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: otel-spinapp
+spec:
+  image: ghcr.io/spinkube/spin-operator/cpu-load-gen:20240311-163328-g1121986
+  executor: containerd-shim-spin
+  replicas: 1
+---
+apiVersion: core.spinoperator.dev/v1alpha1
```

Comment:
I feel like in the documentation we typically have them already install this `containerd-shim-spin` executor. I worry that if they just `k apply -f config/samples/otel.yaml` that they will get an error because it already exists. What if you made this a different executor like `my-custom-otel-shim-executor` (obvs choose a better name).

Reply:
How about `otel-shim-executor`?